docs/content/doc/advanced/config-cheat-sheet.en-us.md

```diff
@@ -15,218 +15,218 @@ menu:
 
 ## Database (`database`)
 
-- `DB_TYPE`: The database type you choose, either `mysql`, `postgres`, `mssql` or `sqlite3`.
-- `HOST`: Database host address and port.
-- `NAME`: Database name.
-- `USER`: Database username.
-- `PASSWD`: Database user password.
-- `SSL_MODE`: For PostgreSQL only.
-- `PATH`: For SQLite3 only, the database file path.
+- `DB_TYPE`: **mysql**: The database type you choose, either `mysql`, `postgres`, `mssql` or `sqlite3`.
+- `HOST`: **127.0.0.1:3306**: Database host address and port.
+- `NAME`: **gitea**: Database name.
+- `USER`: **root**: Database username.
+- `PASSWD`: **\<none\>**: Database user password.
+- `SSL_MODE`: **disable**: For PostgreSQL only.
+- `PATH`: **data/gitea.db**: For SQLite3 only, the database file path.
 
 ## Security (`security`)
 
-- `INSTALL_LOCK`: Indicates whether to allow the open install page (setting admin account is involved, so it's a very important value).
-- `SECRET_KEY`: Global secret key for your server security, **you'd better change it** (will generate a random string every time you install).
-- `LOGIN_REMEMBER_DAYS`: Cookie lifetime, in days.
-- `COOKIE_USERNAME`: Name of the cookie that saves username.
-- `COOKIE_REMEMBER_NAME`: Name of cookie that saves auto-login information.
-- `REVERSE_PROXY_AUTHENTICATION_USER`: Header name for reverse proxy authentication username.
-- `DISABLE_GIT_HOOKS`: Prevent all users (including admin) from creating custom git hooks (defaults to false)
+- `INSTALL_LOCK`: **false**: Disable to allow accessing the install page.
+- `SECRET_KEY`: **random at every install**: Global secret key. This should be changed.
+- `LOGIN_REMEMBER_DAYS`: **7**: Cookie lifetime, in days.
+- `COOKIE_USERNAME`: **gitea\_awesome**: Name of the cookie used to store the current username.
+- `COOKIE_REMEMBER_NAME`: **gitea\_incredible**: Name of cookie used to store authentication information.
+- `REVERSE_PROXY_AUTHENTICATION_USER`: **X-WEBAUTH-USER**: Header name for reverse proxy authentication.
+- `DISABLE_GIT_HOOKS`: **false**: Prevent all users (including admin) from creating custom git hooks.
 
 ## OpenID (`openid`)
 
-- `ENABLE_OPENID_SIGNIN`: Whether to allow signin in via OpenID (defaults to false).
-- `ENABLE_OPENID_SIGNUP`: Whether to allow registering via OpenID (defaults to `!DISABLE_REGISTRATION`).
-- `WHITELISTED_URIS`: Space separated list of POSIX regexp patterns. If non empty OpenID URIs should match any of these to be granted access.
-- `BLACKLISTED_URIS`: Space separated list of POSIX regexp pattenrs. OpenID URI matching any of these is refused access.
+- `ENABLE_OPENID_SIGNIN`: **false**: Allow authentication in via OpenID.
+- `ENABLE_OPENID_SIGNUP`: **! DISABLE\_REGISTRATION**: Allow registering via OpenID.
+- `WHITELISTED_URIS`: **\<none\>**: If non-empty, list of POSIX regex patterns matching OpenID URI's to permit.
+- `BLACKLISTED_URIS`: **\<none\>**: If non-empty, list of POSIX regex pattenrs matching OpenID URI's to block.
```

Comment:
also if empty, would be the empty list.... This is different from `WHITELISTED_URL` in that an empty list would NOT mean to allow NOBODY but rather to allow ALL.

Reply:
Is this just a problem with using the ```<none>```? Would it be better to s/none/empty/?

Reply:
It's a problem with the wording `If non-empty`, which is not needed for `BLACKLISTED_URIS`. There's no difference in behavior when the value is EMPTY. Also please mind the typos